actors/migration/nv9/top.go

```diff
@@ -44,9 +44,30 @@ type Logger interface {
 	Log(level rt.LogLevel, msg string, args ...interface{})
 }
 
+type MigrationCacheKey string
+
+func ActorHeadKey(addr address.Address, head cid.Cid) MigrationCacheKey {
+	return MigrationCacheKey(fmt.Sprintf("%s-h-%s", addr.String(), head.String()))
```

Comment:
Nit: it probably doesn't matter here, but this kind of string formatting is much more expensive than just concatenating them.